Lecture_27/Homework_27.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <title>Резюме</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <style>
+    table {
+        background-color: gray;
+    }
+</style>
+</head>
+<body>
+  <script>
+		var buttonsContainer = document.createElement("div");
+		buttonsContainer.classList.add("text-center", "mx-auto");
+		
+		var addFriendBtn = document.createElement("button");
+		addFriendBtn.innerHTML = "Додати в друзі";
+		addFriendBtn.classList.add("btn", "btn-primary", "btn-round", "btn-lg", "float-left");
+		
+		var messageBtn = document.createElement("button");
+		messageBtn.classList.add("btn", "btn-primary", "btn-round", "btn-lg", "mx-auto");
+		messageBtn.innerHTML = "Написати повідомлення";
+		
+		var jobOfferBtn = document.createElement("button");
+		jobOfferBtn.innerHTML = "Запропонувати роботу";
+		jobOfferBtn.classList.add("btn", "btn-primary", "btn-round", "btn-lg", "float-right");
+
+		document.body.appendChild(addFriendBtn);
+		document.body.appendChild(messageBtn);
+		document.body.appendChild(jobOfferBtn);
```

Comment:
and add buttons to before created container
```suggestion
		buttonsContainer.appendChild(addFriendBtn);
		buttonsContainer.appendChild(messageBtn);
		buttonsContainer.appendChild(jobOfferBtn);
```